domene/src/main/kotlin/no/nav/tiltakspenger/domene/behandling/BehandlingIverksatt.kt

```diff
@@ -31,10 +31,10 @@ data class BehandlingIverksatt(
             )
         } else {
             // todo() her må vi lage en revurdering
-            val nyBehandling = BehandlingOpprettet.fromDb(
+            val nyBehandling = BehandlingOpprettet(
                 id = BehandlingId.random(),
                 sakId = this.sakId,
-                søknader = listOf(this.søknad()),
```

Comment:
Måten flere søknader er håndtert på i løsningen er uansett mangelfull, men denne endringen dropper vel logikken for å hente ut kun siste søknad i dette feltet, ved .søknad(). Ikke sikkert det gjør noe da.

Reply:
Jeg burde ikke tatt med denne endringen i samme PR som det andre, det var dumt av meg. Jeg kan godt ta den ut.

Reply:
Men er det ikke litt rart at hvis man har flere søknader i en behandling, så blir bare en av dem med videre når man oppretter behandlingen i en annen "fase" ?